commons/src/main/java/io/hyscale/commons/constants/ToolConstants.java

```diff
@@ -43,7 +43,7 @@
     
     public static final String LINUX_FILE_SEPARATOR = "/";
 
-    public static final String HYSCALE = "Hyscale";
+    public static final String HYSCALE = "HyScale";
 
```

Comment:
Existing paths get effected.

Reply:
This is only used in version command, for directories we have defined it in SetupConfig.
Will rename it to HYSCALE_CAMELCASE?